### PR TITLE
Bind Dcc wallet info in person details screen (EXPOSUREAPP-11499)

### PR DESCRIPTION
**Pull request by jurajkusnier**
### Testing
- Add any certificate
- Test Menu > CLL Configuration & WalletInfo > Add DCC Wallet Info
- Open person details screen
- The person details screen contains mock data
- The "non-filled shield + checkmark" for VC is removed and not displayed anymore; was previously displayed during the 14-day waiting period (see Jira Story  https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11036)

### Jira Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11499

### CODE REVIEW

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragmentTest.kt [hunk 1]
```diff
@@ -135,21 +135,21 @@ class PersonDetailsFragmentTest : BaseUITest() {
...
-                        twoGCertificate = vaccinationCertificate2,
-                        testCertificate = testCertificate
-                    ),
-                    colorShade = PersonColorShade.COLOR_1
+                    colorShade = PersonColorShade.COLOR_1,
+                    titleText = "Status-Nachweis",
+                    subtitleText = "2G+ PCR-Test",
+                    badgeText = "2G+",
+                    longText = "Ihre Zertifikate erfüllen die 2G-Plus-Regel. Wenn Sie Ihren aktuellen Status vorweisen müssen, schließen Sie diese Ansicht und zeigen Sie den QR-Code auf der Zertifikatsübersicht.",
+                    faqAnchor = "FAQ"
```
Comment (mtwalli) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragmentTest.kt:143]: I think we should provide English too?

Comment (jurajkusnier) [reply Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragmentTest.kt]: updated

**Review by AlexanderAlferov — CHANGES_REQUESTED** [review]

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/text/UrlResource.kt [hunk 2]
```diff
@@ -3,9 +3,14 @@ package de.rki.coronawarnapp.ccl.dccwalletinfo.text
...
 
 fun urlResource(
-    faqAnchor: String,
+    faqAnchor: String?,
     locale: Locale = Locale.getDefault()
 ) = lazy {
-    val lang = if (locale.language == Locale.GERMAN.language) locale.language else Locale.ENGLISH.language
-    "https://www.coronawarn.app/$lang/faq/#$faqAnchor"
+    when {
+        faqAnchor.isNullOrBlank() -> null
```
[the remark this answers is not in the document]
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/text/UrlResource.kt:10]: according to the documentation:
> if `faqAnchor` is set and not empty, a link to the FAQ with the link text shown in Figma and the URL derived from faqAnchor as per CCL > Resolving an FAQ Anchor to a URL ⎆. 
I don't see a use case where we should show FAQ links without an anchor.

**Review by AlexanderAlferov — APPROVED** [review]
Great job! Thanks for test menu.
As for FAQ: if changes required after clarification, I will handle it in my PR.

**Review by SamuraiKek — APPROVED** [review]
LGTM. Nice addition to the test menu ❤️

**Pull request merged**